systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -29,7 +29,7 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
 
-public class StUtils {
+public class StUtils implements Constants {
```

Comment:
I clearly wasn't paying attention when this `Constants` class was added. Having constants on an interface which you inherit is considered a bit of an anti pattern. The more accepted pattern would be a class with a private constructor and using `static imports` at use sites if you don't want to class-qualify the construct name. You don't have to change it in this PR, but another PR to fix it would be appreciated.

Reply:
Good point, I opened https://github.com/strimzi/strimzi-kafka-operator/pull/1608 for fix that. If you have any other concerns about this. feel free to leave comment in new PR, thanks! :)

Reply:
Thanks for that @Frawless.